Dockerfile

```
@@ -1,5 +1,5 @@
 #Copied from https://github.com/dacort/metabase-athena-driver/blob/d7572cd99551ea998a011f8f00a1e39c1eaa59b8/Dockerfile
-ARG METABASE_VERSION=v0.46.6.2
+ARG METABASE_VERSION=v0.50.23
```

```suggestion
ARG METABASE_VERSION=v0.50.26
```

how about we upgrade to the [latest](https://github.com/metabase/metabase/releases/tag/v0.50.26) and greatest

---

@borft I havent tried it for .26, only .25. Can you confirm it works on .26?

---

Have tried yesterday, works fine :)